spec.yml

```diff
@@ -85,6 +85,13 @@ x-resources:
   - name: Shapes
     description: |
       "Shapes are vectors that each user can create and manage. These shapes can then be used in various ways within the Raster Foundry platform."
+  - name: Licenses
+    description: |
+      "A license is an official permission or permit to do, use, or own something."
```

Comment:
I would add something like "Users can add licenses to the data they create and share in Raster Foundry to communicate to others if and how that data can be used".